engine/context/ecs-integration.md

```diff
@@ -33,6 +33,8 @@ To deploy Docker containers on ECS, you must meet the following requirements:
 
 2. Ensure you have an AWS account.
 
+3. Log into the AWS console and opt your AWS account into the new ARN resource ID format. For more information, see [Migrating your Amazon ECS deployment to the new ARN and resource ID format](https://aws.amazon.com/blogs/compute/migrating-your-amazon-ecs-deployment-to-the-new-arn-and-resource-id-format-2/){: target="_blank" class="_"}.

```

Comment:
@metcalfc do you think we need this as a general instruction or has the change been around for long enough that we can have it as part of the FAQ?

Reply:
FAQ is fine.

Reply:
I think if were put the error message and fix in the faq that would be great.

Reply:
> @metcalfc do you think we need this as a general instruction or has the change been around for long enough that we can have it as part of the FAQ?

@chris-crone Do you mean the Desktop FAQ? I don't think we have an FAQ for the partner integration docs yet.

Reply:
I was thinking just for the ECS integration. We could put it on the GitHub repo like [we did for ACI](https://github.com/docker/aci-integration-beta#faq), I'm not sure what makes it visible enough for users.

Reply:
@usha-mandya I think it would be reasonable to just put a heading at the bottom of this doc.

```FAQ

1. What does `this tool requires the "new ARN resource ID format"` mean?

This integration requires the new ARN resource ID format for ECS. To learn more , see [Migrating your Amazon ECS deployment to the new ARN and resource ID format](https://aws.amazon.com/blogs/compute/migrating-your-amazon-ecs-deployment-to-the-new-arn-and-resource-id-format-2/){: target="_blank" class="_"}.. To enable it log into the AWS console and opt your AWS account into the new ARN resource ID format here: https://console.aws.amazon.com/ecs/home?#/settings.

Reply:
@metcalfc Updated. PTAL